kubernetes/services/node_manager.yaml

```diff
@@ -6,7 +6,7 @@ data:
   application.properties: |
     spring.redis.host=localhost
     spring.redis.port=6380
-    ignite.host=10.99.135.32
+    ignite.host=ignite-ip-service.default.svc.cluster.local
```

Comment:
It seems that we are sharing the same Ignite service in this version. Could we rename it to ignite-alcor-service?

Reply:
Sure